File: VB-Examples/03_POP3 Client/WorkingWithMessages/GetMailSummaryInfo.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Email
Imports Spire.Email.Pop3

Namespace GetMailSummaryInfo
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			' Create a POP3 client
			Dim client As New Pop3Client()
			client.Host = Me.HostInput.Text
			client.Username = Me.Username.Text
			client.Password = Me.Password.Text
			client.Port = Integer.Parse(Me.PortInput.Text)
			client.EnableSsl = True
			client.Connect()
			For i As Integer = 1 To client.GetMessageCount()
				Dim msg As MailMessage = client.GetMessage(i)
                MessageBox.Show(String.Format("From:{0}\nTo:{1}\nSubject:{2}", msg.From, msg.To, msg.Subject))
			Next i
		End Sub
	End Class
End Namespace

File: CS-Examples/03_POP3 Client/WorkingWithMessages/GetMailSummaryInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Email;
using Spire.Email.Pop3;

namespace GetMailSummaryInfo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            // Create a POP3 client
            Pop3Client client = new Pop3Client();
            client.Host = this.HostInput.Text;
            client.Username = this.Username.Text;
            client.Password = this.Password.Text;
            client.Port = int.Parse(this.PortInput.Text);
            client.EnableSsl = true;
            client.Connect();
            for (int i = 1; i <= client.GetMessageCount(); i++)
            {
                MailMessage msg = client.GetMessage(i);
                MessageBox.Show(string.Format("From:{0}\nTo:{1}\nSubject:{2}", msg.From, msg.To, msg.Subject));
            }
        }
    }
}